### PR TITLE
[hi] Localize content/hi/cloud-computing.md

### PR DESCRIPTION
**Pull request by shivam-sharma7**
- created a `cloud-computing.md` file and Localise in Hindi who prefer learning and understanding more in the Hindi language can understand better.
 
### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco).

### CODE REVIEW

**Comment by sarkartanmay393** [comment]
You can change the PR's name from `Create cloud-computing.md` to `[hi] Localize cloud-computing.md` or `[hi] Localize content/hi/cloud-computing.md`

**Comment by Garima-Negi** [comment]
@shivam-sharma7 can you join us on slack : cloud-native.slack.com on the #glossary-localization-hindi channel?

**Review by Garima-Negi — CHANGES_REQUESTED** [review]
Hi @shivam-sharma7 the PR looks good, can you also fix the alignment of the sentences? they seem to be on extra newlines and discontinued. thanx!

**Review by Garima-Negi — APPROVED** [review]

**Review by sayantani11 — APPROVED** [review]
Looks good!

**Review by jayesh-srivastava — APPROVED** [review]

File: content/hi/cloud-computing.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+category: अवधारणा
+tags: ["infrastructure", "", ""]
+---
+## यह क्या है ?
+क्लाउड कंप्यूटिंग एक ऐसा मॉडल है जो इंटरनेट पर ऑन-डिमांड सीपीयू, नेटवर्क और डिस्क क्षमताओं जैसे कंप्यूट संसाधन प्रदान करता है।
+क्लाउड कंप्यूटिंग उपयोगकर्ताओं को दूरस्थ भौतिक स्थान में कंप्यूटिंग शक्ति तक पहुंचने और उपयोग करने की क्षमता देता है।
+क्लाउड प्रदाता जैसे AWS, GCP, Azure, DigitalOcean, और अन्य सभी तृतीय पक्षों की पेशकश करते हैं
+एकाधिक भौगोलिक स्थानों में संसाधनों की गणना करने के लिए किराए पर पहुंच की क्षमता।
+
+## समस्या यह संबोधित करता है
```
Comment (anubha-v-ardhan) [inline content/hi/cloud-computing.md:13]:
```suggestion
## समस्या
```

File: content/hi/cloud-computing.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+कंप्यूटिंग शक्ति के अपने उपयोग का विस्तार करने का प्रयास करते समय संगठनों को परंपरागत रूप से दो मुख्य समस्याओं का सामना करना पड़ा।
+वे या तो सुविधाओं का अधिग्रहण, समर्थन, डिजाइन और भुगतान करते हैं अपने भौतिक सर्वर और नेटवर्क को होस्ट करने या उन सुविधाओं का विस्तार और रखरखाव करने के लिए।
+क्लाउड कंप्यूटिंग संगठनों को अपनी कंप्यूटिंग जरूरतों के कुछ हिस्से को किसी अन्य संगठन को आउटसोर्स करने की अनुमति देता है।
+
+## यह कैसे मदद करता है ?
```
Comment (anubha-v-ardhan) [inline content/hi/cloud-computing.md:18]:
```suggestion
## समाधान
```

File: content/hi/cloud-computing.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+क्लाउड प्रदाता संगठनों को ऑन-डिमांड गणना संसाधनों को किराए पर लेने और उपयोग के लिए भुगतान करने की क्षमता प्रदान करते हैं।
+यह दो प्रमुख नवाचारों की अनुमति देता है: संगठन समय बर्बाद किए बिना चीजों को आजमा सकते हैं और नए भौतिक बुनियादी ढांचे पर पैसा या संसाधन खर्च कर सकते हैं
+और वे [पैमाने](/ मापनीयता/) आवश्यकतानुसार और मांग पर कर सकते हैं। क्लाउड कंप्यूटिंग संगठनों को अपनी जरूरत के अनुसार अधिक या कम बुनियादी ढांचे को अपनाने की अनुमति देता है।
```
Comment (anubha-v-ardhan) [inline content/hi/cloud-computing.md:21]:
```suggestion
और वे आवश्यकतानुसार [स्केल](/scalability/) कर सकते हैं। क्लाउड कंप्यूटिंग संगठनों को अपनी जरूरत के अनुसार अधिक या कम बुनियादी ढांचे को अपनाने की अनुमति देता है।
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
LGTM! Thankyou @shivam-sharma7 for your contributions.
Sorry for the extremely long time this review took

**Comment by shivam-sharma7** [comment]
> LGTM! Thankyou @shivam-sharma7 for your contributions. Sorry for the extremely long time this review took

No problem! 🙂

**Review by anubha-v-ardhan — APPROVED** [review]

**Comment by anubha-v-ardhan** [comment]
Thank You @shivam-sharma7

**Pull request merged**